CHARTER.adoc

```diff
@@ -0,0 +1,195 @@
+
+Please feel free to forward this announcement to any other appropriate lists. The OGC is an open standards organization; we encourage your feedback.
+
+== Purpose of the Standards Working Group
+
+The GeoZarr Standard Working Group (SWG) is chartered to develop a Zarr encoding for geospatial gridded data in the form of Zarr conventions (based on the approach described in the draft Zarr Enhancement Proposal 4).  Zarr specifies a protocol and format used for storing Zarr arrays, while GeoZarr defines **conventions** and recommendations for storing **multidimensional georeferenced grid** of geospatial observations (including rasters). The GeoZarr SWG will also work on improving the Climate and Forecast (CF) conventions if necessary, particularly for alternative coordinate reference system (CRS) encoding if relevant.
+
+
+
+== Business Value Proposition
+
+In the geospatial world, new cloud native data formats are emerging. Zarr is a generic data format for n-dimensional arrays that enables access to data in compressed chunks of the original array and has become increasingly popular to use for geospatial purposes. In June 2022, the OGC endorsed a community standard of Zarr V2.0 (https://zarr.readthedocs.io/en/stable/spec/v2.html). The purpose of this charter is to adopt a more explicit GeoZarr as an OGC Standard which would provide guidance to standardize the approach for encoding various aspects of geospatial data in zarrs.
+
+== Scope of Work
+
+The goal of the GeoZarr specification is to establish flexible and inclusive conventions for the Zarr cloud-native format, specifically designed to meet the diverse requirements within the geospatial domain. These conventions aim to provide a clear and standardized framework for organizing and describing data, ensuring unambiguous representation. 
+
+The objectives of GeoZarr conventions includes:
+
+1. Compatibility: Ensuring easy compatibility with popular mapping and data analysis tools such as GDAL, Xarray, ArcGIS, and QGIS, enabling seamless integration into existing workflows.
+2. Visualisation: Simplifying the creation and display of geospatial data in web browsers without the need for complex workarounds, making geospatial information more accessible to users.
+3. Dimensions: Supporting multidimensional data, such as hyperspectral and altitude information, to address diverse geospatial data requirements.
+4. Data Discovery: Providing metadata for discovering, accessing, and retrieving the data, including subsets or different arrangements of the data.
+5. Mixing Data: Facilitating the combination of different types of geospatial data, including satellite images, elevation maps, and weather models, to create comprehensive and informative datasets.
+8. Flexibilty: Allowing scientists and researchers to work with diverse data types and projections in their preferred software and programming languages, promoting flexibility and adaptability in geospatial data processing and analysis.
```

Comment:
I fought (and lost) the battle to explicitly _exclude_ visualization from the OGC GeoDataCube SWG charter scope. But I'm going to try again!

In my opinion, everything after objective (1) is downstream of objective (1). I.e., I might consider simplifying this to something like:

> The overarching objective of the GeoZarr conventions is to provide a single unified specification for spatially-referenced data that simultaneously supports existing tooling and workflows for working with multi-dimensional gridded data (e.g., Xarray, NetCDF libraries) and existing tooling and workflows for efficient analysis if georeferenced raster and vector data (e.g., GDAL). This will enable GeoZarr datasets to be seamlessly integrated into existing workflows and tools for interactive analysis and visualization by a variety of users.

Reply:
I strongly disagree, as it is a key objective of cloud-native data formats (similar as COG). Furthermore, achieving this objective does not require significant effort or complexity.

Reply:
I would support keeping visualization in scope. Visual interpretation of these data is fundamental to any development activity and should be a first class use case.

Reply:
I have taken out visualization as it's own item, instead nodding to it under compatibility. Also I have changed the wording of the following paragraph to use the terminology conformance classes. I think this will get us un-stuck and allow for further discussions within the SWG.  

> Specifically, the convention should provide guidance to standardize the approach for encoding various aspects of geospatial data, including, for example the following list of potential conformance classes: 

> Part of the effort of this working group is to determine what should be kept as a part of the GeoZarr core requirements, and what aspects of geospatial data should be kept as separate conformance classes, which may or may not be applicable to specific sub-domains of the geospatial community.